static/js/services/translation.js

```diff
@@ -58,27 +58,27 @@ define(["jed", "underscore"], function(Jed, _) {
 		// _n
 		this._n = _.bind(function() {
 			if (domain && context) {
-				return _.bind(function(singular, plural) {
-					var vars = Array.prototype.slice.call(arguments, 1);
-					var r = i18n.translate(singular).onDomain(domain).withContext(context).ifPlural(vars[0], plural);
+				return _.bind(function(num, singular, plural) {
+					var vars = Array.prototype.slice.call(arguments, 3);
```

Comment:
I incorrectly read the code used in ifPlural, saw `arguments` instead of `vars` which wouldn't work with the ifPlural api `ifPlural( num, plural_key )`